### ISSUE TITLE
sqlite example incorrect?

### ISSUE THREAD

**Issue by slava-vishnyakov** [markbates/pop#13]
This is from README.md file:

```
staging:
  dialect: "sqlite3"
  url: "./staging.sqlite"
```

Running it, I get:

```
Unknown dialect !
```

however in `connection_details.go` there is this:

```
func (cd *ConnectionDetails) Finalize() error {
	if cd.URL != "" {
		u, err := url.Parse(cd.URL)
		if err != nil {
			return errors.Wrapf(err, "couldn't parse %s", cd.URL)
		}
		cd.Dialect = u.Scheme
```

which basically overwrites the `dialect:` from file with `u.Scheme`, resulting in 

```
		cd.Dialect = ""
```
and later:

```
Unknown dialect !
```

(note the empty dialect name)

It seems to me that the one way would be to use:

```
staging:
  url: sqlite://staging.sqlite
```

Alas... then we have this line:

https://github.com/markbates/pop/blob/master/sqlite.go#L28

Which results in file called `?_busy_timeout=5000` (yep, with a question mark) being created..
and `staging.sqlite` is nowhere to be found.

Thanks!

**Comment by slava-vishnyakov** [comment]
Finally, after some digging, the example, probably should be:

```
staging:
  dialect: sqlite3
  database: ./staging.sqlite
```

**Comment by slava-vishnyakov** [comment]
Added a Pull request to fix the docs #14
